client/src/router/index.js

```diff
@@ -5,6 +5,7 @@ import qs from 'qs';
 import UserDataProvider from '@/components/guard/UserDataProvider';
 import LoginGuard from '@/components/guard/LoginGuard';
 import AuthGuard from '@/components/guard/AuthGuard';
+import RoutersTesterGuard from '../components/guard/RoutersTesterGuard';
```

Comment:
rename `RoutersTesterGuard` to `UserExistGuard` or smth like that

Reply:
In the future, I should check if :nickname/:id  For this I should create another Guard?

Reply:
Maybe is, like EventTypeExistGuard

Reply:
made renaming maybe proposed name more meaning for developers